src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -224,58 +220,7 @@ export const NavBar: React.FC = track(
               </NavItem>
             </NavSection>
 
-            <NavSection>
-              {isLoggedIn && (
-                <>
-                  <NavItem
-                    href="/works-for-you"
-                    Menu={NotificationsMenu}
-                    Overlay={NotificationsBadge}
-                    onClick={() => {
-                      trackEvent({
-                        action_type: AnalyticsSchema.ActionType.Click,
-                        subject: AnalyticsSchema.Subject.NotificationBell,
-                        new_notification_count: getNotificationCount(),
-                        destination_path: "/works-for-you",
-                      })
-                    }}
-                  >
-                    {({ hover }) => {
-                      if (hover) {
-                        trackEvent({
-                          action_type: AnalyticsSchema.ActionType.Hover,
-                          subject: AnalyticsSchema.Subject.NotificationBell,
-                          new_notification_count: getNotificationCount(),
-                        })
-                      }
-                      return <BellIcon fill={hover ? "purple100" : "black80"} />
-                    }}
-                  </NavItem>
-                  {conversationsEnabled && (
-                    <NavItem href="/user/conversations">
-                      {({ hover }) => {
-                        return (
-                          <EnvelopeIcon
-                            fill={hover ? "purple100" : "black80"}
-                          />
-                        )
-                      }}
-                    </NavItem>
-                  )}
-                  <NavItem Menu={UserMenu}>
-                    {({ hover }) => {
-                      if (hover) {
-                        trackEvent({
-                          action_type: AnalyticsSchema.ActionType.Hover,
-                          subject: "User",
-                        })
-                      }
-                      return <SoloIcon fill={hover ? "purple100" : "black80"} />
-                    }}
-                  </NavItem>
-                </>
-              )}
-            </NavSection>
+            <NavSection>{isLoggedIn && <LoggedInActions />}</NavSection>
```

Comment:
Is the empty `<NavSection>` when not logged in useful or maybe this can be
```tsx
{isLoggedIn && (
  <NavSection>
    <LoggedInActions />
  </NavSection>
)}
```